examples/kaniko-local/skaffold.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: skaffold/v1alpha3
+kind: Config
+build:
+  artifacts:
+  - imageName: gcr.io/k8s-skaffold/skaffold-example
+  kaniko:
+    buildContext: 
+      localDir: true
```

Comment:
maybe this should just be a struct? `localDir: {}`? 

specifying `localDir: false` doesn't really make sense

Reply:
+1, if you make these pointers to structs in the config, the `oneOf` yamltag will do implicit validation for you.